kubernetes_deploy/development/migrate-db-job.yml

```diff
@@ -22,4 +22,3 @@ spec:
             value: laalaa
           - name: DB_HOST
             value: laa-legal-adviser-api-shared-services
-
```

Comment:
Was this on purpose?

Reply:
Yeah, @sldblog spotted it in the previous PR but we decided to merge and fix it in this one.
The trailing blank line was only ever a pasting error